packages/node/src/integrations/http.ts

```diff
@@ -192,13 +192,28 @@ function _createWrappedRequestMethodFactory(
 
       const scope = getCurrentHub().getScope();
 
+      const spanData: Record<string, string> = {
+        url: requestUrl,
+        method: requestOptions.method || 'GET',
+      };
+      if (requestOptions.hash) {
+        // strip leading "#"
```

Comment:
Hmm, there is some inconsistency in our own docs.
https://develop.sentry.dev/sdk/data-handling/ vs https://develop.sentry.dev/sdk/performance/span-data-conventions/

@AbhiPrasad @antonpirker WDYT? `?foo=bar` or `foo=bar`, `#foo` or `foo´.

Reply:
Let's make it `foo=bar` for the query and `foo` for the fragment

this matches what most language std lib uses (`urlparse` in python, `parse_url` in PHP)

Reply:
We do not use `parse_url` and actually already shipped with `?` and `#`. If we want to remove it, I can PR once the docs are updated.

Reply:
Let's go ahead and change the docs!

Reply:
Confirming, it sounds like I should leave this as is?

Reply:
Yep, we're good for this PR!